JULABO/iocBoot/iocJULABO-IOC-01/config.xml

```diff
@@ -4,6 +4,10 @@
 <ioc_desc>Julabo water bath</ioc_desc>
 <macros>
 <macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM Port" />
+<macro name="READ_POWER_CMD_NUMBER" pattern="^[0-9]+$" description="Command number for reading power. The prefix 'IN_PV_' is added automatically. Defaults to 01" />
```

Comment:
Can the macros have the same order as the rows in the spreadsheet linked to on the Julabo wiki page? This would make it easier to not configure the Julabo incorrectly

Reply:
The GUI will re-order them alphabetically anyway - the order in this file doesn't make any difference

Reply:
As discussed, I have added a ticket to set the ordering explicitly: https://github.com/ISISComputingGroup/IBEX/issues/3796